ReactWindows/ReactNative.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
We can do this in a separate PR.

Reply:
doesn't CodePush use the ReactPage type as a way to hook in via reflection and do its thing? @pre10der89 brought this up some months ago as a barrier to refactoring in this area.

Reply:
@matthargett your memory is correct... The code push module does take the react page type as a dependency to gain access to the concrete type react instance...

Reply:
It's not a firm barrier. CodePush will continue to work for older versions of RNW. For newer versions, it will need to be updated. I'll try to make sure the CodePush update happens before this lands in an official version.

Reply:
This will actually make the CodePush implementation more straightforward.